doc/docker.md

```diff
@@ -1,16 +1,102 @@
 # Running with Docker
 
+The advantage of using docker-compose is that it is configured to run several services that the Control Panel relies on, such as the PostgreSQL database and Redis. It initializes these ready for local development purposes.
+
+## Build
+
 To build the docker image, use the following command:
+
 ```sh
 docker build -t controlpanel .
 ```
 
-Running the app requires a PostgreSQL database, and this is provided with
-docker-compose:
+## .env file
+
+You need to create a `.env` file with the settings to enable it to connect to external services.
+
+You probably want it to pick up your 'data' AWS profile:
+```sh
+AWS_PROFILE=data
+```
+
+For testing purposes, you can simply connect to some services used for the dev environment. To do this, for the keys listed below, copy the matching lines from chart-env-config/dev/cpanel.yml and just change the format of each line to `KEY=value`.
```

Comment:
I don't have a `chart-env-config/dev/cpanel.yml` file in my repository. Where can I find it..?

Reply:
I've added directions to this file now